tests/python/dump_lbs.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+""" Display the contents of a kivaloo LBS server. """
+
+import kivaloo.servers
+import kivaloo.proto_lbs
+
+
+def main():
+    """ Dump the contents of a LBS server to stdout. """
+    lbs = kivaloo.servers.Server_lbs()
+    proto_lbs = kivaloo.proto_lbs.Proto_lbs()
+
+    # Find which blocks are available
+    _, next_block, last_block = proto_lbs.params2()
+    if last_block == 2**64 - 1:
+        print("No data in the LBS server.")
+        lbs.shutdown()
+        exit(0)
+    print("lbs has next_block %i, last_block %i" % (next_block, last_block))
+
+    # Print data from blocks
+    print("------")
+    for i in range(last_block + 1):
+        status, data = proto_lbs.get(i)
+        if status == 0:
+            print("block %i:" % i)
+            print(data)
```

Comment:
Dumb question, does `print` do something reasonable with binary data?

Reply:
Depends on your definition of "reasonable"?
```
td@mac: ~/src/kivaloo/tests/python (python-lbs)
$ ./test_proto_lbs.py 
td@mac: ~/src/kivaloo/tests/python (python-lbs)
$ ./dump_lbs.py 
lbs has next_block 1, last_block 0
------
block 0:
b'\x00\x00\x00\x04\x00\x00\x00
... repeat for 512 blocks, minus however many bytes are already here...
\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
------
```

I'm not intending on using `dump_lbs.py` for anything serious (unless something goes *drastically* wrong).  I'm much more interested in `dump_kvlds.py`, which of course isn't in this PR, but is thematically similar.

Reply:
Yeah, by "reasonable" I meant "not spewing unprintable characters onto the console".

Reply:
BTW I just finished writing `kvlds-dump` and I'm about to write `kvlds-undump`.  I decided that these would be useful utilities to have...

Reply:
ok.  Well, if nothing else, the python script is an easily-understandable example of how to use the library.